packages/dev/core/src/Animations/animatable.ts

```diff
@@ -342,20 +344,20 @@ export class Animatable {
                 }
 
                 if (runtimeAnimations.length == 0) {
-                    this._scene._activeAnimatables.splice(idx, 1);
+                    if (!useGlobalSplice) {
+                        this._scene._activeAnimatables.splice(idx, 1);
+                    }
                     this._raiseOnAnimationEnd();
                 }
             }
         } else {
             const index = this._scene._activeAnimatables.indexOf(this);
 
             if (index > -1) {
-                this._scene._activeAnimatables.splice(index, 1);
-                const runtimeAnimations = this._runtimeAnimations;
-
-                for (let index = 0; index < runtimeAnimations.length; index++) {
-                    runtimeAnimations[index].dispose();
+                if (!useGlobalSplice) {
+                    this._scene._activeAnimatables.splice(index, 1);
                 }
+                this._runtimeAnimations = [];

```

Comment:
We don't dispose the runtime animations anymore?

Reply:
They only remove themselves from the runtimeAnimations so it is a shortcut

Reply:
Perhaps we should put a comment in the `RuntimeAnimation.dispose` method to indicate that no "real" disposal should be done here? For example, if `RuntimeAnimation` allocated an object that should be disposed of in `dispose`, we would have a memory leak by not calling the method.

Reply:
true. I will add it